src/layouts/components/Homepage/InfobarBody.tsx

```diff
@@ -89,7 +89,7 @@ export const InfobarBody = ({
       </FormControl>
       <Button
         id={`section-${index}`}
-        onClick={() => onDelete(`section-${index}`, "Infobar Section")}
```

Comment:
can i get context on why did we delete this?

Reply:
The modal already renders section. 
Rn in prod we repeat section twice 
<img width="821" alt="Screenshot 2023-09-20 at 7 12 28 PM" src="https://github.com/isomerpages/isomercms-frontend/assets/42832651/e408a219-308a-46ea-ab16-7f5623249c7e">

Reply:
good catch!